packages/st2/debian/install

```diff
@@ -6,3 +6,4 @@
 ../st2actions/conf/syslog.*.conf etc/st2/
 ../st2reactor/conf/logging.*.conf etc/st2/
 ../st2reactor/conf/syslog.*.conf etc/st2/
+../conf/nginx usr/share/doc/st2/conf/
```

Comment:
thie target directory is `/usr/share/doc/st2/conf/nginx`. we copy directory nginx to directory `doc/st2/conf/`...